virttest/qemu_vm.py

```diff
@@ -4309,6 +4309,36 @@ def check_block_locked(self, value):
                     return block['locked']
         return False
 
+    def check_block_opened(self, value):
```

Comment:
better add 'tray' in the function name, like check_cdrom_tray_opened.
It may confused with storage migration open status.